test/mpi/dtpools/src/Makefile.am

```diff
@@ -4,7 +4,7 @@
 #     See COPYRIGHT in top-level directory.
 #
 AM_CPPFLAGS = $(MPI_H_INCLUDE)
-AM_CPPFLAGS += -fvisibility=default
+#AM_CPPFLAGS += -fvisibility=default
```

Comment:
We should use the existing macros to add visibility flags when available.

Reply:
Why do you need fvisibility flag for tests? I had to remove that because compiler on Windows does not support this flag.